### PR TITLE
windows lolbin 'gathernetworkinfo.vbs' detection

### PR DESCRIPTION
**Pull request by blueteamer8699**
…for information gathering. Based on the information provided in: https://posts.slayerlabs.com/living-off-the-land/#gathernetworkinfovbs

### CODE REVIEW

File: rules/windows/process_creation/win_lolbin_cscript_gathernetworkinfo.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+logsource:
+    category: process_creation
+    product: windows
+    service: sysmon
+detection:
+    selection1:
+        CommandLine|contains: 'cscript.exe'
+    selection2:
+        CommandLine|contains: 'gatherNetworkInfo.vbs'
+    condition: selection1 AND selection2
```
Comment (frack113) [inline rules/windows/process_creation/win_lolbin_cscript_gathernetworkinfo.yml:26]: `condition: selection1 and selection2`

File: rules/windows/process_creation/win_lolbin_cscript_gathernetworkinfo.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+author: blueteamer8699
+date: 2022/01/03
+modified: 2021/03/01
+tags:
+    - attack.discovery
+    - attack.group_policy_discovery
+    - attack.execution
+    - attack.command_and_scripting_interpreter
+    - attack.visual_basic
+    - attack.t1059_005
```
Comment (frack113) [inline rules/windows/process_creation/win_lolbin_cscript_gathernetworkinfo.yml:16]: `attack.t1059.005`

File: rules/windows/process_creation/win_lolbin_cscript_gathernetworkinfo.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+title: Gather information about a system using gatherNetworkInfo.vbs script
```
Comment (frack113) [inline rules/windows/process_creation/win_lolbin_cscript_gathernetworkinfo.yml:1]: `has a title that has not title capitalization. Words: 'information, about, system, using, script'`

File: rules/windows/process_creation/win_lolbin_cscript_gathernetworkinfo.yml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    - attack.discovery
+    - attack.group_policy_discovery
+    - attack.execution
+    - attack.command_and_scripting_interpreter
+    - attack.visual_basic
+    - attack.t1059_005
+logsource:
+    category: process_creation
+    product: windows
+    service: sysmon
```
Comment (frack113) [inline rules/windows/process_creation/win_lolbin_cscript_gathernetworkinfo.yml:20]: `service: sysmon` is to remove as the rule use category.
sysmon and EventID are add when convert with the /config/generic/sysmon.yml file in the sigmac option

**Review by frack113 — CHANGES_REQUESTED** [review]
Hi , you can use `python tests/test_rules.py` to check you rule.

**Review by manasmbellani — APPROVED** [review]
updates have been made in subsequent commits.

**Comment by manasmbellani** [comment]
Please see commit 2 in this pull request. All the changes have been applied.
Btw,  I am not sure why I am unable to see the same feedback when I run the python3 script.
`$ python3 /opt/sigma/tests/test_rules.py
[taxii2client.v20] [WARNING ] [2022-01-03 18:06:20,633] TAXII Server Response did not include 'Content-Range' header - results could be incomplete.
[taxii2client.v20] [WARNING ] [2022-01-03 18:06:20,673] TAXII Server Response with different amount of objects! Setting per_request=707
[taxii2client.v20] [WARNING ] [2022-01-03 18:06:22,370] TAXII Server Response did not include 'Content-Range' header - results could be incomplete.
[taxii2client.v20] [WARNING ] [2022-01-03 18:06:22,372] TAXII Server Response with different amount of objects! Setting per_request=72
[taxii2client.v20] [WARNING ] [2022-01-03 18:06:25,273] TAXII Server Response did not include 'Content-Range' header - results could be incomplete.
[taxii2client.v20] [WARNING ] [2022-01-03 18:06:25,281] TAXII Server Response with different amount of objects! Setting per_request=474
[taxii2client.v20] [WARNING ] [2022-01-03 18:06:27,201] TAXII Server Response did not include 'Content-Range' header - results could be incomplete.
[taxii2client.v20] [WARNING ] [2022-01-03 18:06:27,204] TAXII Server Response with different amount of objects! Setting per_request=133
...............................
----------------------------------------------------------------------
Ran 31 tests in 0.005s

OK`

**Review by frack113 — APPROVED** [review]

**Pull request merged**